client/command/generate.go

```diff
@@ -814,3 +851,76 @@ func displayCanaries(canaries []*clientpb.DNSCanary, burnedOnly bool) {
 		}
 	}
 }
+
+// GenerateUniqueIP generates and returns an available IP which can then
+// be assigned to a Wireguard interface
+func GenerateUniqueIP() (net.IP, error) {
+	peersTunIps, err := db.WGPeerIPs()
```

Comment:
This is apparently the only place the `db` package is used. Please replace that logic with a dedicated RPC call to retrieve the peer IPs.

Reply:
Done as above